### DISCUSSION TITLE
V5 文档建议

### DISCUSSION THREAD

**Discussion by fxzer** [antvis/G6#6173] [General]
比如：

1. 自定义节点那边的模块，看完好几遍，都不知道怎么下手；建议增加几个自定义节点的示例。

2. 一些 API 解释太过简洁了，比如 graph.setData /updateData之类的，看完会有这样的疑惑：设置完数据图会自动渲染呢 还是 要手动调用 render 进行重新渲染呢？

3. Element - 元素 - xxx，案例代码有经常运行不出来，展示的是```createGraph(....)```代码块，我看到的时候突然一愣：怎么又来了个 `createGraph`，那`new Graph`和这个我到底该用哪个，后来才发现是案例没渲染出来的原因。

4. 虽然 GUI 上显示了  `html.style.innerHTML` 设置这个属性，但是这让人会有疑惑：是在 `node.style.innerHTML` 中还是在`html.style.innerHTML` 设置该属性值，去翻了其他几个 Element 图形才能确定在 `node.style.innerHTML` 中设置，建议：还是贴一下关键性代码比较好，减轻阅读者的心智负担。

**Comment by Aarebecca** [comment]
感谢你的建议，
1. API 文档的描述我们会尽快补充相关提示。
2. createGraph 是官网文档中插入的示例所使用的方法，偶发无法执行的问题，会尽快修复
